Tasks/MobileCenterDistribute/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -1,7 +1,7 @@
 {
-  "loc.friendlyName": "Mobile Center Upload",
+  "loc.friendlyName": "Mobile Center Distribute",
   "loc.helpMarkDown": "For help with this task, visit the Visual Studio Mobile Center [support site](https://intercom.help/mobile-center/).",
-  "loc.description": "Upload mobile app packages to Visual Studio Mobile Center",
+  "loc.description": "Distribute mobile app packages to Visual Studio Mobile Center",
```

Comment:
Distribute mobile app packages via Visual Studio Mobile Center? The idea is to distribute the build to your testers/users via Mobile Center